RELEASE.md

```diff
@@ -0,0 +1,119 @@
+3. Promoting RCs to stable
+
+## Cutting release branches
+
+Release branches are typically cut from the branch [`viable/strict`](https://github.com/pytorch/pytorch/tree/viable/strict) as to ensure that tests are passing on the release branch.
+
+Release branches *should* be prefixed like so:
+```
+release/{MAJOR}.{MINOR}
+```
+
+An example of this would look like:
+```
+release/1.8
+```
+
+### Making release branch specific changes
+
+These are examples of changes that should be made to release branches so that CI can function normally on
+them:
+
+* Update target determinator to use release branch:
+  * Example: https://github.com/pytorch/pytorch/pull/40712
+* Cutting a release branch on [`pytorch/xla`](https://github.com/pytorch/xla)
+  * Example: https://github.com/pytorch/pytorch/pull/40721
+* Update backwards compatibility tests to use RC binaries instead of nightlies
+  * Example: https://github.com/pytorch/pytorch/pull/40706
+
+## Drafting RCs (Release Candidates)
+
+To draft RCs a user with necessary permission can push a git tag to the main `pytorch/pytorch` git repository.
+
+The git tag for a release candidate must follow the following format:
+```
+v{MAJOR}.{MINOR}.{PATCH}-rc{RC_NUMBER}
+```
+
+An example of this would look like:
+```
+v1.8.1-rc1
+```
+
+Pushing a release candidate should trigger the `binary_builds` workflow within CircleCI using [`pytorch/pytorch-probot`](https://github.com/pytorch/pytorch-probot)'s [`trigger-circleci-workflows`](trigger-circleci-workflows) functionality.
+
+This trigger functionality is configured here: [`pytorch-circleci-labels.yml`](https://github.com/pytorch/pytorch/blob/master/.github/pytorch-circleci-labels.yml)
+
+### Release Candidate Storage
+
+Release candidates are currently stored in the following places:
+
+* Wheels: https://download.pytorch.org/whl/test/
+* Conda: https://anaconda.org/pytorch-test
+* Libtorch: https://download.pytorch.org/libtorch/test
+
+Backups are stored in a non-public S3 bucket at [`s3://pytorch-backup`](https://s3.console.aws.amazon.com/s3/buckets/pytorch-backup?region=us-east-1&tab=objects)
+
+### Cherry Picking Fixes
+
+Typically within a release cycle fixes are necessary for regressions, test fixes, etc.
+
+For fixes that are to go into a release after the release branch has been cut we typically employ the use of a cherry pick tracker.
+
+An example of this would look like:
+* https://github.com/pytorch/pytorch/issues/51886
+
+**NOTE**: The cherry pick process is not an invitation to add new features, it is mainly there to fix regressions
+
+## Promoting RCs to Stable
+
+Promotion of RCs to stable is done with this script:
+[`pytorch/builder:release/promote.sh`](https://github.com/pytorch/builder/blob/master/release/promote.sh)
+
+Users of that script should take care to update the versions necessary to the versions you are attempting to promote for the specific packages you are attempting to promote.
```

Comment:
```suggestion
Users of that script should take care to update the versions necessary for the specific packages you are attempting to promote.
```
I actually am a lil confused by what this sentence is saying and I don't know if my suggestion helped